build/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+# We need sudo before all
+RUN apt-get -qy update && \
+    apt-get -qy install sudo
+
+# Add jenkins user
+RUN useradd -s /bin/bash jenkins
+RUN echo "jenkins    ALL=(ALL)       NOPASSWD: ALL">> /etc/sudoers
```

Comment:
this violates the security policy

Reply:
Yes, but it's better than running all commands as `root` even inside the container.